client/src/Tools/_framework/Paths/Activities.tsx

```diff
@@ -671,70 +714,101 @@ export function Activities() {
       ) : null}
       <Flex
         data-test="Activities"
-        padding="10px"
+        padding="0 10px"
         margin="0px"
         width="100%"
-        justifyContent="center"
-        background="var(--lightBlue)"
+        background={listView ? "white" : "var(--lightBlue)"}
         minHeight="calc(100vh - 120px)"
```

Comment:
With moving the tile buttons to the top row, the header got significantly taller, so `100vh - 120px` so no longer the right calculation for the amount of screen space left. One gets a scrollbar even before the contents need it. By trial and error, it seems that `100vh - 192px` is the largest you can make it without having the scrollbar appear. But, you can wait until you've finished designing the header to determine what this minimum height should be.